ui/StatusQ/src/StatusQ/Components/StatusExperimentalTag.qml

```diff
@@ -0,0 +1,23 @@
+import QtQuick 2.13
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Rectangle {
+    id: root
+
+    implicitHeight: 20
+    implicitWidth: 36
+    radius: 4
+    color: "transparent"
+    border.width: 1
+    border.color: Theme.palette.baseColor1
+
+    StatusBaseText {
+        id: label
+        font.pixelSize: 11
+        font.weight: Font.Medium
+        color: Theme.palette.baseColor1
+        anchors.centerIn: parent
+        text: "Beta"
```

Comment:
I guess it's intended not to be translatable, right? 😄 

Otherwise we'd probably need to use a `Control` with this rectangle as background and the text as contentItem to provide some dynamic width.

Reply:
Indeed. Beta is a software term, I don't think it makes sense to translate it.

Reply:
Then maybe `StatusBetaTag.qml` as it's set in stone? 😄
Minor thing, really. Your call 😄

Reply:
Yeah, it was originally Beta, then someone made a call to say "Experimental" instead, so I renamed, then it went back to "Beta" and at that point I was like "I ain't renaming again if it is to change again" :laughing: 
Now that it seems set in stone, I can rename sure.